src/adapter/src/catalog/open.rs

```diff
@@ -1535,6 +1534,60 @@ fn add_new_builtin_cluster_replicas_migration(
     Ok(())
 }
 
+/// Roles can have default values for session variables, e.g. you can set a Role default for the
+/// 'cluster' session variable.
+///
+/// This migration exists to remove the Role default for a session variable, if the persisted input
+/// is no longer valid. For example if we remove a session variable or change the accepted set of
+/// values.
+fn remove_invalid_session_var_role_defaults_migration(
```

Comment:
```suggestion
/// Roles can have default values for configuration parameters, e.g. you can set a Role default for the
/// 'cluster' configuration parameter.
///
/// This migration exists to remove the Role default for a configuration parameter, if the persisted input
/// is no longer valid. For example if we remove a configuration parameter or change the accepted set of
/// values.
fn remove_invalid_config_param_role_defaults_migration(
```

Reply:
I think we want to refer to them as configuration parameters everywhere now, though now that I look at it, the main struct is still called `SessionVars`.

Reply:
done!